docs/dev/table/hive/index.md

```diff
@@ -92,6 +92,34 @@ to make the integration work in Table API program or SQL in SQL Client.
 Alternatively, you can put these dependencies in a dedicated folder, and add them to classpath with the `-C`
 or `-l` option for Table API program or SQL Client respectively.
 
+Apache Hive is built on Hadoop, so you need Hadoop dependency first, please refer to
+[Providing Hadoop classes]({{ site.baseurl }}/ops/deployment/hadoop.html#providing-hadoop-classes).
+
+There are two way to hive dependencies. First way, using bundled hive jar, providing multiple bundle
+hive jars that can cover all remote metastore versions. Second way, user defined dependencies, you
+can build your own dependencies if you need to. 
+
+#### Using bundled hive jar
+
+The following tables list all available bundled hive jars. You can pick one to the `/lib/` directory in Flink distribution.
+
+{% if site.is_stable %}
+
+| Remote Metastore   | Dependency Version  | Maven dependency             | SQL Client JAR         |
```

Comment:
I don't think the `Dependency Version` column is useful. We just need to let users know which jar to download (or which mvn dependency to use), given the version of their HMS.